### PR TITLE
[W-8885816] OFM-NPSP: Automerge major release branches

### PR DESCRIPTION
**Pull request by spelak-salesforce**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done

Refer to [Definition of Done](https://salesforce.quip.com/9P7hAOPHJJyU) to see any additional details for the items below:
- ~Any net new LWC work has JEST test coverage 50% or above~
- ~Default Sa11y tests pass for all LWC components~
- ~🔒 Secure both Front-end (LWC) & back-end (Apex) as necessary~
- ~🔑 Grant users access in Permission Sets (Object, Field, Apex Class) as necessary~
- [x] Link the pull request and work item by PR comment and Chatter post respectively, e.g. GUS: [W-0000000: Work Name]()
- [x] Make sure that ACs are updated (if any gaps)
- [x] **All acceptance criteria have been met**
    - [x] Developer
    - [x] Code Reviewer
- ~Pull Request contains draft release notes~
- ~Labels, help text, and customer facing messages are reviewed by Docs~
- ~QE story level testing completed~

### CODE REVIEW

**Comment by spelak-salesforce** [comment]
[W-8885805](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B00000096VvwIAE/view)

File: cumulusci.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ project:
     name: OutboundFundsNPSP
     package:
         name: Outbound Funds (npsp)
-        namespace: outfundsnpspext
+        namespace: lh6uuas
```
Comment (seanpat09) [inline cumulusci.yml:5]: Did you mean to change the naemspace?

File: cumulusci.yml [hunk 2]
```diff
@@ -49,6 +54,23 @@ tasks:
             sql_path: datasets/data.sql
 
 flows:
+    dependencies:
+        steps:
+            1:
+                task: update_dependencies
+                options:
+                    include_beta: False
```
Comment (seanpat09) [inline cumulusci.yml:62]: Was this to get around the NPSP "mismatched versions" issue we were running into or would we prefer not to use the beta?

**Review by seanpat09 — CHANGES_REQUESTED** [review]

File: cumulusci.yml (hunk 1 above)
Comment (spelak-salesforce) [reply cumulusci.yml:5]: Ooops!   Great catch.  That was from a test.

File: cumulusci.yml (hunk 2 above)
Comment (spelak-salesforce) [reply cumulusci.yml:62]: Ooops! Great catch. That was from a test.

File: cumulusci.yml (hunk 1 above)
Comment (spelak-salesforce) [reply cumulusci.yml]: @seanpat09 Fixed.

File: cumulusci.yml (hunk 2 above)
Comment (spelak-salesforce) [reply cumulusci.yml]: @seanpat09 Fixed.

**Comment by spelak-salesforce** [comment]
@jstvz , can you give a codeowner review?  I think we still need to wait for the NPSP package version on Spring '21 issue to be resolved before all the feature builds run succcessfully.

**Comment by davidmreed** [comment]
The feature builds should all succeed. MetaCI is pinned to Spring '21 pods until after R2, which happens late tonight.

**Review by seanpat09 — APPROVED** [review]
nice work!

**Comment by spelak-salesforce** [comment]
@SalesforceFoundation/release-engineering-reviewers , can we get a code owner review?  Thank you!

**Review by jstvz — APPROVED** [review]

**Pull request merged**